about.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Code Coop</title>
+    <link rel="stylesheet" href="assets/css/about.css">
+  </head>
+  <body>
+    <header>
+        <h1>Code Coop</h1>
+    </header>
+      <nav>
+        <a href="index.html" title="Home page">About</a>
+        <a href="contact.html" title="Contact page with contact details">Contact Us</a>
+        <a href="#">Other</a>
+      </nav>
```

Comment:
Should the `<nav>` element go into the `<header>` or `<main>` section?

Reply:
Latest commit has removed these links as they were just a placeholder.

Reply:
I've placed the `<nav>` inside of the `<header>` as I think it will make more sense.